snippets/card-product.liquid

```diff
@@ -216,7 +217,7 @@
     </div>
   </div>
 {%- else -%}
-  <div class="product-card-wrapper card-wrapper underline-links-hover">
+  <div class="card-wrapper underline-links-hover {{ wrapper_class }}">
```

Comment:
This is what my product card looks when no product is selected:
<img width="407" alt="image" src="https://user-images.githubusercontent.com/37168033/182604702-5202490e-a0a2-4390-99be-3340cb9f4b04.png">
And this is what it looks like if I select a product:
<img width="402" alt="image" src="https://user-images.githubusercontent.com/37168033/182604887-6cacc5ae-23f5-4f34-b0aa-fdfe515cd6b5.png">

These are the settings:
<img width="288" alt="image" src="https://user-images.githubusercontent.com/37168033/182604812-0385f84e-4dd2-4588-99fb-2e585cb43fbd.png">

So it looks like we're missing a `product-card-wrapper` class here, because only the color is being applied when using the empty state.

I think we should revert back to what it was before?
```suggestion
  <div class="product-card-wrapper card-wrapper underline-links-hover">
```

Reply:
Good catch! I only updated the top one